test/testcafe/spec/AuthenticatorYubikeyView_spec.js

```diff
@@ -29,13 +29,9 @@ const verifyMock = RequestMock()
   .onRequestTo('http://localhost:3000/idp/idx/challenge/answer')
   .respond(xhrSuccess);
 
-async function setup(t, widgetOptions) {
-  const options = widgetOptions ? { render: false } : {};
+async function setup(t) {
   const pageObject = new YubiKeyAuthenticatorPageObject(t);
-  await pageObject.navigateToPage(options);
-  if (widgetOptions) {
-    await renderWidget(widgetOptions);
```

Comment:
Old usage of `renderWidget` to update the widget options was unreliable and caused a lot of flakiness and failed tests when I was making the changes in this PR. I updated things to override the options _before_ the widget loads/bootstraps for the first time instead.